ci/taos/plugins-good/pr-format-nobody.sh

```diff
@@ -67,7 +66,6 @@ function pr-format-nobody(){
         else
             echo "[DEBUG] commit body checker is PASSED. patch file name: $filename"
             echo "[DEBUG] current directory is `pwd`"
-            check_result="success"
```

Comment:
This line is kept to make sure the final result value. Anyway. It is okay. Let's update that.